engine/src/main/java/org/terasology/rendering/dag/nodes/BackdropNode.java

```diff
@@ -56,27 +56,27 @@
     public void initialise() {
         super.initialise();
         playerCamera = worldRenderer.getActiveCamera();
-        requiresFBO(new FBOConfig(REFRACTIVE_REFLECTIVE, FULL_SCALE, FBO.Type.HDR).useNormalBuffer(), displayResolutionDependentFBOs);
+        requiresFBO(new FBOConfig(REFRACTIVE_REFLECTIVE, FULL_SCALE, FBO.Type.HDR, displayResolutionDependentFBOs).useNormalBuffer());
     }
 
     @Override
     public void process() {
         PerformanceMonitor.startActivity("rendering/backdrop");
 
-        initialClearing();
-
-        READ_ONLY_GBUFFER.bind();
-        READ_ONLY_GBUFFER.setRenderBufferMask(true, true, true);
-
-        playerCamera.lookThroughNormalized();
-        /**
+        /*
          * Sets the state to render the Backdrop. At this stage the backdrop is the SkySphere
          * plus the SkyBands passes.
          *
          * The backdrop is the only rendering that has three state-changing methods.
          * This is due to the SkySphere requiring a state and the SkyBands requiring a slightly
          * different one.
          */
```

Comment:
This was classified as a dangling comment, hence the move by a few lines and the demotion from javadoc to simple block comment.